src/Aacotroneo/Saml2/Saml2User.php

```diff
@@ -47,14 +47,14 @@ function getAttributes()
      */
     function getRawSamlAssertion()
     {
-        return Input::get('SAMLResponse'); //just this request
+        return input('SAMLResponse'); //just this request
     }
 
     function getIntendedUrl()
     {
-        $relayState = Input::get('RelayState'); //just this request
+        $relayState = input('RelayState'); //just this request
 
-        if ($relayState && URL::full() !=$relayState) {
+        if ($relayState && url()->full() != $relayState) {
```

Comment:
@beeglebug we have this issue #31 this works on Laravel 5.2 only. Sorry to ask to you but Im not using Laravel right now. What can we do here? For versions <5.2 mainly. (I was so used to Facades :) )